l1-contracts/contracts/state-transition/chain-deps/ZkSyncHyperchainStorage.sol

```diff
@@ -150,4 +163,8 @@ struct ZkSyncHyperchainStorage {
     uint128 baseTokenGasPriceMultiplierDenominator;
     /// @dev The optional address of the contract that has to be used for transaction filtering/whitelisting
     address transactionFilterer;
+    /// @dev
+    SyncLayerState syncLayerState;
```

Comment:
Should we not store this in the Bridgehub, in a mapping?

Reply:
I mean, the chain should be aware of its state in mailbox